scripts/scripts.js

```diff
@@ -108,6 +130,9 @@ async function loadLazy(doc) {
 
   loadCSS(`${window.hlx.codeBasePath}/styles/lazy-styles.css`);
   loadFonts();
+
+  window.hlx.plugins.run('loadLazy');
+  import('../tools/sidekick/aem-experimentation.js');
```

Comment:
I would only do this if the sidekick is loaded. So I'd hook into the sidekick event here